wasm/lib/src/convert.rs

```diff
@@ -13,6 +15,9 @@ use rustpython_vm::{exceptions, py_serde};
 use crate::browser_module;
 use crate::vm_class::{stored_vm_from_wasm, WASMVirtualMachine};
 
+// Currently WASM do not support multithreading. We should change this once it is enabled.
+thread_local!(static JS_FUNC: RefCell<Option<js_sys::Function>> = RefCell::new(None));
```

Comment:
Maybe this could be a vec instead, so multiple js functions can be passed to python? And then you could do something like:
```rust
let idx = JS_FUNC.with(|funcs| {
    let mut funcs = funcs.borrow_mut();
    funcs.push(func);
    funcs.len() - 1
});
```
and then capture `idx` in a closure.

Reply:
The problem is that we can't use closures because they are `!Send+!Sync`. Maybe I can create a new type of method that has a `idx` as well.

Reply:
Are you sure? https://doc.rust-lang.org/reference/types/closure.html#other-traits

Reply:
> Are you sure? https://doc.rust-lang.org/reference/types/closure.html#other-traits

Sure I am wrong. Thanks :pray: